libs/ui/src/Prose.tsx

```diff
@@ -0,0 +1,75 @@
+import styled from 'styled-components'
+
+interface Props {
+  compact?: boolean
+  maxWidth?: boolean
+  textCenter?: boolean
+}
+
+export const Prose = styled('div')<Props>`
+  margin: ${({ textCenter }) => (textCenter ? 'auto' : undefined)};
+
+  /* width: 100%; */
+  max-width: ${({ maxWidth = true }) => (maxWidth ? '66ch' : undefined)};
+  text-align: ${({ textCenter }) => (textCenter ? 'center' : undefined)};
+  line-height: 1.2;
+  @media (min-width: 480px) {
+    line-height: 1.3;
+  }
+  & h1 {
+    margin: 2rem 0 1rem;
```

Comment:
I've noticed before that election-manager (and maybe other apps) version of this code has "em" everywhere instead of "rem", is rem what we want to be using?

Reply:
No. I need to swap to `em` in the Prose component. LMK if you'd like know more about where we use `em` vs `rem`.